skynet/modules/ttt/processor.py

```diff
@@ -150,19 +203,29 @@ async def summarize(payload: DocumentPayload, job_type: JobType, model: BaseChat
     return formatted_result
 
 
-async def process_open_ai(payload: DocumentPayload, job_type: JobType, api_key: str, model_name=None) -> str:
+async def process_open_ai(
+    payload: DocumentPayload, job_type: JobType, api_key: str, model_name=None, customer_id: str | None = None
+) -> str:
     llm = ChatOpenAI(
         api_key=api_key,
         max_completion_tokens=payload.max_completion_tokens,
         model_name=model_name,
         temperature=0,
     )
 
+    if job_type == JobType.ASSIST:
```

Comment:
I feel like we could avoid doing the same logic if we'd make these take a function as a parameter, some `process_func`.

We can do that at a later time.

That would also mean that we can run on OCI to process too, which this PR doesn't account for, for example.